public/app/core/components/TagFilter/TagValue.tsx

```diff
@@ -15,7 +15,7 @@ export class TagValue extends React.Component<Props, any> {
     this.onClick = this.onClick.bind(this);
   }
 
-  onClick(event: React.SyntheticEvent) {
+  onClick(event: React.SyntheticEvent<HTMLDivElement>) {
```

Comment:
maybe should update this to `event: React.MouseEvent<HTMLDivElement>`

Reply:
ah thanks, somehow missed that 😕